src/OrchardCore.Themes/TheAdmin/Assets/js/menu.js

```diff
@@ -18,6 +18,19 @@ $('.leftbar-compactor').click(function () {
     $('body').hasClass('left-sidebar-compact') ? unSetCompactStatus() : setCompactStatus(true);
 });
 
+$('#left-nav li.has-items').click(function () {
```

Comment:
We should remove the call to hoverintent at the bottom of the file.

Reply:
Please add commits on top of that branch if there's anything. I just wanted to have this in a working state at least for beta 2.